src/data/markdown/translated-guides/en/06 Testing Guides/04 Running large tests.md

```diff
@@ -78,11 +79,11 @@ const checkRes = check(res, {
 });
 ```
 
-Code like this runs fine when the system under test (SUT) is not overloaded and returns proper responses.
+Code like this runs fine when the [system under test](/misc/glossary/#system-under-test) (SUT) is not overloaded and returns proper responses.
 When the system starts to fail, the above check won't work as expected.
 
-The issue here is that the check assumes that there's always a body in a response. The `r.body` may not exist if server is failing.
-In such case, the check itself won't work as expected and error similar to the one below will be returned:
+The issue here is that the check assumes that there's always a body in a response. The `r.body` may not exist if the server is failing.
+In such case, the check itself won't work as expected and an error similar to the one below will be returned:
```

Comment:
Shouldn't we add an example of how to handle this exception? Right now it seems we diagnose a problem, but don't give any way to find a solution.

Reply:
Oh, I realize now that this is in the text, just not in the diffs. Still, it looks like the example only fails a check if there is a body to begin with.

```javascript
const checkRes = check(res, {
  'Homepage body size is 11026 bytes': (r) => r.body && r.body.length === 11026,
});
```

Wouldn't it make more sense to have two checks? One that checks that a body is returned and other that, when returned, it's the correct length?

Reply:
Matt, I think you're nitpicking at this point. It's fine as is :)